cypress/integration/settings/users/owners.spec.js

```diff
@@ -20,7 +20,7 @@ describe('Management of n fee/fine owners and service points', () => {
       // clarify should be service points be shared between existing users
       cy.allure().startStep('Check presented owners and related service points');
       UsersOwners.getUsedServicePoints().then(usedServicePoints => {
-        addedServicePoints.push(UsersOwners.defaultServicePoints.filter(servicePoint => !usedServicePoints?.includes(servicePoint))[0]);
+        addedServicePoints.push(...UsersOwners.defaultServicePoints.filter(servicePoint => !usedServicePoints?.includes(servicePoint)));
         cy.allure().endStep();
```

Comment:
In future will be better if we put allure marks inside fragments instead tests

Reply:
xz